code/game/machinery/gulag_teleporter.dm

```diff
@@ -30,7 +30,8 @@ The console is located at computer/gulag_teleporter.dm
 		/obj/item/clothing/head/helmet/space/plasmaman,
 		/obj/item/tank/internals,
 		/obj/item/clothing/mask/breath,
-		/obj/item/clothing/mask/gas))
+		/obj/item/clothing/mask/gas,
+		/obj/item/restraints/handcuffs))
```

Comment:
um 👀

Reply:
I mean, this was already the case, wasn't it?

Reply:
Line 140 of the old code

Reply:
sorry I was enjoying my weekend.

Can I ask how you are reaching that conclusion? Old L140-142 states that the handcuff moves to the top of the gulag teleporter (so you get your cuffs back).

Reply:
....oh